IFC4x3/PropertyConstants/o/ONTHEFIELD_0wE3oCwo58swPTd_8371Uo.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ONTHEFIELD_0wE3oCwo58swPTd_8371Uo" Name="ONTHEFIELD" UniqueId="3a383c8c-eb21-48db-a65d-9fe2031c17b2" />
+
```

Comment:
Definition needed?

Reply:
Done. Commit : b9da2b65d5470974f60d4ecfa44ab16135543152

Reply:
Was this also renamed?